hat/assets/js/apps/Iaso/domains/orgUnits/reviewChanges/Dialogs/ReviewOrgUnitGroupChanges.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FunctionComponent } from 'react';
+import {
+    Table,
+    TableBody,
+    TableCell,
+    Accordion,
+    AccordionSummary,
+    AccordionDetails,
+    TableRow,
+} from '@mui/material';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { NestedGroup } from '../types';
+import { NewOrgUnitField } from '../hooks/useNewFields';
+
+type Props = {
+    groups: NestedGroup[];
+    newAddedGroups: NestedGroup[];
+    status: string | undefined;
+    field: NewOrgUnitField;
+};
+
+export const ReviewOrgUnitGroupChanges: FunctionComponent<Props> = ({
+    groups,
+    newAddedGroups,
+    status,
+    field,
+}) => {
+    const isCellApproved = (status && status === 'approved' && 'success.light') || '';
+    const isSelected = (status && field?.isSelected === true && "success.light") || "";
+    
+    return (
+        <Accordion>
```

Comment:
can you try to use this one: hat\assets\js\apps\Iaso\components\Accordion please ?
It is the same but i revisited the style a bit